recipes/quickstart/NotebookLlama/README.md

```diff
@@ -0,0 +1,82 @@
+## NotebookLlama: An Open Source version of NotebookLM
+
+[Listen to audio from the example here](./resources/_podcast.mp3)
+
+This is a guided series of tutorials/notebooks that can be taken as a reference or course to build a PDF to Podcast workflow. 
+
+You will also learn from my experimentation of using Text to Speech Models.
+
+It assumes zero knowledge of LLMs, prompting and audio models, everything is covered in their respective notebooks.
+
+### Outline:
+
+Here is step by step thought (pun intended) for the task:
+
+- Step 1: Pre-process PDF: Use `Llama-3.2-1B` to pre-process and save a PDF
+- Step 2: Transcript Writer: Use `Llama-3.1-70B` model to write a podcast transcript from the text
+- Step 3: Dramatic Re-Writer: Use `Llama-3.1-8B` model to make the transcript more dramatic
+- Step 4: Text-To-Speech Workflow: Use `parler-tts/parler-tts-mini-v1` and `bark/suno` to generate a conversational podcast
+
+### Detailed steps on running the notebook:
+
+Requirements: GPU server or an API provider for using 70B, 8B and 1B Llama models.
+
+Note: For our GPU Poor friends, you can also use the 8B and lower models for the entire pipeline. There is no strong recommendation. The pipeline below is what worked best on first few tests. You should try and see what works best for you!
+
+- First, please Install the requirements from [here]() by running inside the folder:
+
+```
+git clone https://github.com/meta-llama/llama-recipes
+cd llama-recipes/recipes/quickstart/NotebookLlama/
+pip install -r requirements.txt
+```
+
+- Notebook 1:
+
+This notebook is used for processing the PDF and processing it using the new Feather light model into a `.txt` file.
+
+Update the first cell with a PDF link that you would like to use. Please decide on a PDF to use for Notebook 1, it can be any link but please remember to update the first cell of the notebook with the right link. 
+
+Please try changing the prompts for the `Llama-3.2-1B-Instruct` model and see if you can improve results.
```

Comment:
how do we ensure the LLM is not changing the text/ keep the fidelity of the original PDF?

Reply:
I will add it here in the ReadMe outline as well-please see the prompt for the 1B model. It's prompted to not change/summarise